chain/chain/src/test_utils.rs

```diff
@@ -334,13 +334,13 @@ impl RuntimeAdapter for KeyValueRuntime {
         Ok(validators[offset + delta].account_id.clone())
     }
 
-    fn get_num_missing_blocks(
+    fn get_num_validator_blocks(
         &self,
         _epoch_id: &EpochId,
         _last_known_block_hash: &CryptoHash,
         _account_id: &AccountId,
-    ) -> Result<u64, Error> {
-        Ok(0)
+    ) -> Result<(u64, u64), Error> {
```

Comment:
What does this pair of numbers mean?

P.S. Well, going through the code I read that they are "num_blocks_produced" and "num_blocks_expected", but I would like to ask for a better design if possible as otherwise, contributors spend a ton of time lurking through the code to learn how something is used instead of what that was meant to be in the first place.